README.md

```diff
@@ -31,6 +31,25 @@ csq <filename>
 -   Don't commit two different things together.
 -   After all things push your code to our **main branch**.
 
+
+## Development Steps
+
+### 1. Data Types
```

Comment:
Already datatypes are there like STR, FLOAT and LIST
And the boolean is expressed in terms of FLOAT only so please see it.